lib/active_admin/filters/active_filter.rb

```diff
@@ -26,19 +26,29 @@ def values
       end
 
       def label
+        translated_predicate = translated_predicate_name.mb_chars.downcase.to_s
```

Comment:
@deivid-rodriguez we support down to Ruby 2.2. Are there any issues in 2.2 and 2.3?